tests/integration/iop-cni-defaults.yaml

```diff
@@ -0,0 +1,25 @@
+# This file provides CNI defaults for integration testing.
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  components:
+    cni:
+      namespace: kube-system
+      enabled: true
+      k8s:
+        env:
+          - name: LOG_UDS_ADDRESS
```

Comment:
Why not the default? Is this just for testing until its stable?

Reply:
Yeah, more testing, and I also thought this will need a value option, which I want to avoid at least for now. But look again, we could just hard code the log volume. If user want to tune the path, they can edit the overlay and env variable as wanted. I've updated it to be enabled by default.